CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog 
 
+## Unreleased
+
+- breaking change: update_device_chats() was removed. This is now done automatically during configure.
```

Comment:
nitpicking: usually, we are referring to the ffi-functions here, that would be dc_update_device_chats(), however, thanks for already adding a hint here, this is very helpful for the one writing the next changelog :)